internal/table/scanner/scanner.go

```diff
@@ -11,6 +11,9 @@ import (
 
 	"github.com/ydb-platform/ydb-go-genproto/protos/Ydb"
 
+	"github.com/ydb-platform/ydb-go-sdk/v3/internal/decimal"
+	scanner2 "github.com/ydb-platform/ydb-go-sdk/v3/internal/scanner"
```

Comment:
Why do you need import alias for scanner?

Reply:
for exclude circle importing